io/pci/EEH.py

```diff
@@ -105,11 +106,19 @@ def test_eeh_basic_pe(self):
         enter_loop = True
         num_of_miss = 0
         num_of_hit = 0
-        while num_of_hit <= self.max_freeze:
+        while num_of_hit < self.max_freeze:
```

Comment:
@maramsmurthy it should be `<=` only because the driver fails only on nth time eeh trigger, if logic is with `<` the it will chek for only n-1 time and driver is expected to re-cover all these time.
so keep the original one

Reply:
@Naresh-ibm I hope we can read this parameter from OS and increment the value by 1 directly instead of passing it in yaml. Intention of this test is full filled only after exceeding supporting number of EEH hits. 
Can I make the changes as suggested above. If so I will add it in setup method itself.

Reply:
@maramsmurthy this is a mandatory yaml parameter. if any one wants to go for eeh stress, like more than 100 times and see if the driver is recovering, they can always change the /sys dir parameter value to 100 and run this test case. That is what test is doing. it will read the max value from yaml and set it to that partiular value and trigger those many times.